src/Sylius/Behat/Page/Checkout/CheckoutPaymentStep.php

```diff
@@ -25,6 +25,10 @@ public function selectPaymentMethod($paymentMethod)
     {
         $radio = $this->getDocument()->findField($paymentMethod);
 
+        if (null === $radio) {
+            throw new \RuntimeException('Payment method not found or it is not visible');
```

Comment:
Is there any more appropriate exception for missing/not visible element? Also, missing "." at the end of exception message.